Description: Removed all references to rubygems
 Removed all references to rubygems as per
 https://wiki.debian.org/Teams/Ruby/RubyExtras/UpstreamDevelopers

--- ruby-hiera-eyaml-2.0.5.orig/lib/hiera/backend/eyaml/commands.rb
+++ ruby-hiera-eyaml-2.0.5/lib/hiera/backend/eyaml/commands.rb
@@ -1,4 +1,3 @@
-require 'rubygems'
 
 class Hiera
   module Backend
--- ruby-hiera-eyaml-2.0.5.orig/lib/hiera/backend/eyaml/plugins.rb
+++ ruby-hiera-eyaml-2.0.5/lib/hiera/backend/eyaml/plugins.rb
@@ -1,4 +1,3 @@
-require 'rubygems'
 
 class Hiera
   module Backend
